start-jupyter.sh

```diff
@@ -1,8 +0,0 @@
-#!/usr/bin/env bash
-
-# becuase you're going to forget something this simple
```

Comment:
love this.
trying to add a reivew comment, but "the appropriate button" isn't very specific.

Reply:
how cool. No emails yet and there's a resolve button below.